src/models/addressBook/index.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import {
+  addStringNoLocale,
+  addUrl,
+  createSolidDataset,
+  createThing,
+  getSolidDataset,
+  getSourceUrl,
+  getThing,
+  getUrl,
+  saveSolidDatasetAt,
+  setThing,
+} from "@inrupt/solid-client";
+import { acl, dc, foaf, rdf, schema, vcard } from "rdf-namespaces";
+import { joinPath } from "../../stringHelpers";
+import { chain } from "../../solidClientHelpers/utils";
+import { ERROR_CODES, isHTTPError } from "../../error";
+
+/**
+ * @typedef AddressBook
```

Comment:
I'd prefer not to sprinkle typedoc through the codebase inconsistently.

Reply:
I wanted to use it in models as a reference, to lessen the chance of inconsistent types.

Reply:
I can of course not use JSDoc syntax if you prefer? But I still want some documentation in the "head" of each model module.